deploy/vagrant/bin/init.py

```diff
@@ -13,10 +15,19 @@
 import sys
 from sys import stderr
 import argparse
+import json
+import tempfile
+import errno
+
+import boto
+from boto import ec2
+import yaml
 
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('provider')
+    parser.add_argument('--is-aws-spot', action='store_true')
```

Comment:
it is only for aws, this script should only be called by `create`. 

Yes, It's a bit messy, I think I should split them out to standalone scripts. thanks.